src/collection/layout.js

```diff
@@ -15,6 +15,22 @@ var elesfn = ({
 
     layout.animations = [];
 
+    var calculateSpacing = function( spacing, nodes, pos ){
```

Comment:
`calculateSpacing( spacing, nodesBb, pos )` would be a more efficient signature to avoid unnecessary bb calcs